tsfile/src/main/java/org/apache/iotdb/tsfile/write/schema/MeasurementSchema.java

```diff
@@ -98,12 +98,11 @@ public static MeasurementSchema deserializeFrom(InputStream inputStream) throws
 
     measurementSchema.measurementId = ReadWriteIOUtils.readString(inputStream);
```

Comment:
Add one UT of the serialize and deserialize method

Reply:
OK, I'll add it